[2024-11-08T18:15:04.889+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-08T18:15:04.979+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:14:00+00:00 [queued]>
[2024-11-08T18:15:05.041+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:14:00+00:00 [queued]>
[2024-11-08T18:15:05.042+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-08T18:15:05.156+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_dependencies> on 2024-11-08 18:14:00+00:00
[2024-11-08T18:15:05.198+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=877) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-08T18:15:05.217+0000] {standard_task_runner.py:72} INFO - Started process 880 to run task
[2024-11-08T18:15:05.205+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection', 'load_dependencies', 'scheduled__2024-11-08T18:14:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_dag.py', '--cfg-path', '/tmp/tmp6bdohw8l']
[2024-11-08T18:15:05.218+0000] {standard_task_runner.py:105} INFO - Job 203: Subtask load_dependencies
[2024-11-08T18:15:05.531+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-08T18:14:00+00:00 [running]> on host 641085defe8e
[2024-11-08T18:15:05.991+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fraud_team' AIRFLOW_CTX_DAG_ID='fraud_detection' AIRFLOW_CTX_TASK_ID='load_dependencies' AIRFLOW_CTX_EXECUTION_DATE='2024-11-08T18:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-08T18:14:00+00:00'
[2024-11-08T18:15:05.993+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-08T18:15:06.019+0000] {fraud_detection_dag.py:26} INFO - Chargement des dépendances depuis S3...
[2024-11-08T18:16:05.300+0000] {fraud_detection_dag.py:42} INFO - Modèle chargé avec succès
[2024-11-08T18:16:06.798+0000] {fraud_detection_dag.py:47} INFO - ETL chargé avec succès
[2024-11-08T18:16:06.886+0000] {python.py:240} INFO - Done. Returned value was: True
[2024-11-08T18:16:06.988+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-08T18:16:06.990+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fraud_detection, task_id=load_dependencies, run_id=scheduled__2024-11-08T18:14:00+00:00, execution_date=20241108T181400, start_date=20241108T181504, end_date=20241108T181606
[2024-11-08T18:16:07.146+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-08T18:16:07.255+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-08T18:16:07.260+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
